ingest-app/src/cmr/ingest/data/memory_db.clj

```diff
@@ -239,6 +249,23 @@
                                                                     (count failed-granules)
                                                                     (count skipped-granules)
                                                                     (count task-granules)))))))))))
+
+  (cleanup-bulk-granule-tasks-by-provider
```

Comment:
No need to make this change. The cleanup should only be tested in external db mode.